docs/doxygen-user/installation.dox

```diff
@@ -5,68 +5,28 @@
 \section prereqs Prerequisites
 It is _highly_ recommended to remove or disable any antivirus software from computers that will be processing or reviewing cases. Antivirus software will often conflict with forensic software, and may quarantine or even delete some of your results before you get a chance to look at them.
 <br><br>
+
+
+\section install Deployment Types
+Starting with Autopsy 4.0.0, there are two ways to deploy Autopsy:
```

Comment:
4.0 or 4.0.0? So far, I've made the path be 4.0. Shall I change them to be 4.0.0?